20190519-WebAbstract/BDA_FS19_WebAbstract.tex

```diff
@@ -81,24 +85,34 @@
 \setlength\tabcolsep{0cm}
 
 % Table itself
-\begin{Large}
+\begin{large}
 \noindent
 \begin{tabularx}{\textwidth}{p{5cm}X}
-	\textbf{Themenbereiche:} & \\
+	\textbf{Themenbereiche:} & RFID, Automatisierte Arbeitsprozesse, Machbarkeitsstudie\\
 	\textbf{Studierende:} & Pascal Baumann, Dane Wicki\\
 	\textbf{Betreuungsperson:} & Martin Jud\\
 	\textbf{Experte:} & Urs Gehrig\\
-	\textbf{Auftraggebende:} & Speicherbibliothek Luzern\\
-	\textbf{Keywords:} & RFID, Machbarkeitsstudie\\
+	\textbf{Auftraggebende:} & Verein Kooperative Speicherbibliothek Schweiz\\
+	\textbf{Keywords:} & Logistik, RFID, Machbarkeitsstudie, Suchalgorithmus\\
 \end{tabularx}
-\end{Large}
+\end{large}
 \egroup
 
 \section{Aufgabenstellung}
+Im Hochregallager der Speicherbibliothek werden momentan bis zu 110'000 Behältern mit verschiedenen Exemplaren (von welchen viele mit RFID Tags ausgestattet sind) gelagert. Die Behälter werden manuell von Menschen befüllt und anschliessend wird der Behälter voll autonom an einen Lagerplatz gefahren. Zeitweise können auch gewisse Exemplare wieder aus den Behältern entnommen werden um dieses Exemplar zu Lesen, Scannen oder einer der teilnehmenden Bibliotheken zurückzusenden. Während dem Vorgang des Lagerns und Entnehmen der Exemplare werden weiterhin Menschen für das Befüllen und Pflegen der Behälter verwendet. Dies birgt die Gefahr, dass eine Person aus Versehen ein Exemplar in einen falschen Behälter einsortiert. Und so das Exemplar nur sehr umständlich wiedergefunden werden kann.
 
-\section{Ergebnisse}
+Mit dieser Arbeit soll "mittels einer Machbarkeitsstudie und einem Proof of Concept untersucht werden wie technisch realisiert werden kann, solche mit RFID-ausgerüstete Einzel\-exemplare (Bücher, Zeitschriften, etc.) im vollautomatischen Behälter-Hochregallager mit bis zu 3.1 Mio. Exemplaren zu identifizieren und zu finden".
```

Comment:
Mit dieser Arbeit soll, "mittels einer Machbarkeitsstudie und einem Proof of Concept**,** untersucht werden, wie **????** technisch realisiert werden kann 
ab hier versteh ich nur noch Bahnhof....., solche mit RFID-ausgerüstete Einzel\-exemplare (Bücher, Zeitschriften, etc.) im vollautomatischen Behälter-Hochregallager mit bis zu 3.1 Mio. Exemplaren zu identifizieren und zu finden".